worker/src/RTC/WebRtcTransport.cpp

```diff
@@ -1058,7 +1058,12 @@ namespace RTC
 		}
 
 		// Trick for clients performing aggressive ICE regardless we are ICE-Lite.
-		this->iceServer->ForceSelectedTuple(tuple);
+		if (
+		  this->iceServer->GetState() == RTC::IceServer::IceState::CONNECTED ||
+		  this->iceServer->GetState() == RTC::IceServer::IceState::COMPLETED)
+		{
+			this->iceServer->MayForceSelectedTuple(tuple);
+		}
```

Comment:
You can just call `MayForceSelecteTuple()` as it will return if the state is not correct. There is otherwise a duplication of conditions in the caller and the target method.

Reply:
done